android/app/build.gradle

```diff
@@ -130,7 +130,7 @@ android {
         applicationId "com.mattermost.rnbeta"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 278
+        versionCode 279
```

Comment:
Version bump in a feature PR?

Reply:
It id set to the same version that wxists in master, not sure why is that here... is should not be a problem though